sdf/1.7/camera.sdf

```diff
@@ -151,6 +151,24 @@
       <element name="cy" type="double" default="120" required="1">
         <description>Y principal point (in pixels)</description>
       </element>
+      <element name="p_fx" type="double" default="277" required="0">
```

Comment:
Thank you for adding the elements! I'm wondering if it's best to put these inside a new `<projection>` element that's a sibling of `intrinsics`. Or maybe a child of `intrinsics`? I'd have to refresh my computer vision knowledge.

Reply:
yeah, I don't know enough about these parameters to have a useful opinion

Reply:
> Thank you for adding the elements! I'm wondering if it's best to put these inside a new `<projection>` element that's a sibling of `intrinsics`. Or maybe a child of `intrinsics`? I'd have to refresh my computer vision knowledge.

IIUC the projection matrix is the cumulative effect of intrinsic and extrinsic parameters. So unless a xml element can have two parents (which I don't think is possible) I think @chapulina's suggestion to put these inside a new sibling `<projection>` element is best. I've done this in https://github.com/gazebosim/sdformat/commit/de4e5362bcd75140af26e1d9c2e43a7a11d5c2c8

References
- https://courses.cs.washington.edu/courses/cse576/17sp/notes/CamerasStereo17.pdf
- https://en.wikipedia.org/wiki/Camera_matrix